website/integrations/services/node-red/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Node-RED
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Node-RED
+
+From https://nodered.org/
+
+:::note
+Node-RED is a programming tool for wiring together hardware devices, APIs and online services in new and interesting ways.
+
+It provides a browser-based editor that makes it easy to wire together flows using the wide range of nodes in the palette that can be deployed to its runtime in a single-click.
+:::
+
+:::warning
+This requires modification of the Node-RED settings.js and installing additional Passport-js packages, see [Securing Node-RED](https://nodered.org/docs/user-guide/runtime/securing-node-red#oauthopenid-based-authentication) documentation for further details.
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `authentik.company` is the FQDN of authentik.
+-   `nodered.company` is the FQDN of nodered.
+
+### Step 1
+
+In authentik, create an OAuth2/OpenID Provider (under Resources/Providers) with these settings:
+
+:::note
+Only settings that have been modified from default have been listed.
+:::
+
+-   Name: Node-RED
+-   Slug: node-red
+-   Provider: Node-RED
+
+**Protocol Settings**
+```
+Name: nodered
```

Comment:
Duplicate with above and the setting is not under protocol settings

Reply:
Good point, I copied that error from the Gitea documentation that I modified for this. I'll open a separate pull request to fix that as well in a bit.